AndroidAnnotations/androidannotations-core/androidannotations-test/src/main/java/org/androidannotations/test/eview/ViewWithInstanceState.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (C) 2016 the AndroidAnnotations project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed To in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.androidannotations.test.eview;
+
+import org.androidannotations.annotations.EView;
+import org.androidannotations.annotations.InstanceState;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.view.View;
+
+@EView
+public class ViewWithInstanceState extends View {
```

Comment:
It is enough, if you can check if the state in the superclass is properly saved, however i think you cannot do that. Maybe you can extend from another `View` class, which saves state we can assert against, but the most clean would be writing a non-AA custom `View`.